### PR TITLE
Home page - so many things.

### PR DESCRIPTION
**Pull request by cayla-c**
Hello all,

I hope I've merged allllll the new things y'all have done correctly!  Man that was a lot.  I do not recommend missing 3 days of work.  NO STARS.  Please let me know if something is amiss!!!  And thank you so much.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -51,6 +51,10 @@ module.exports = {
         test: /.(eot|svg|ttf|woff|woff2)$/,
         use: [{ loader: 'file-loader?name=font/[name]__[hash:base64:5].[ext]' }],
       },
+      {
+        test: /.(eot|svg|ttf|woff|woff2)$/,
+        use: [{ loader: 'file-loader?name=font/[name]__[hash:base64:5].[ext]' }]
+      }
     ],
```
Comment (mattBman23) [inline webpack.config.js:58]: Is this a mistake? Seems like duplicates

File: client/dist/style.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  height: 226px;
+  left: 1205px;
+  top: 128px;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: #000000;
+}
```
Comment (ShinanSun) [inline client/dist/style.css:202]: Wondering if this whole css files located in the dist folder. I think the gitignore, ignore everything inside the dist folder... there is a style.css outside the dist folder.

File: webpack.config.js (hunk 1 above)
Comment (cayla-c) [reply webpack.config.js:58]: lol probably a mistake, yeah - I'm scared of config files!

**Review by mattBman23 — APPROVED** [review]
Not sure if i'm the right person to approve, but besides the structure, everything looks good.

File: client/components/homePage/BOTM.jsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { Container, Image, Button, Cards } from 'react-bootstrap';
+import dummyData from "./dummyData.js";
+
+const BOTM = () => {
```
Comment (jfleming9357) [inline client/components/homePage/BOTM.jsx:5]: IMO the name of this function should be more descriptive. I have no idea what it stands for.

File: client/style.css [hunk 4]
```diff
@@ -47,3 +47,206 @@ body {
   width: 100%;
   text-align: center;
 }
+
```
Comment (jfleming9357) [inline client/style.css:50]: Something in this file is breaking images for the rest of the site

File: webpack.config.js (hunk 1 above)
Comment (jfleming9357) [reply webpack.config.js:58]: Yeah 54-57 should be removed

**Review by jfleming9357 — CHANGES_REQUESTED** [review]
Just a couple of fixes needed. Mainly whatever in the css file is breaking the images on the rest of the site and removing the duplicate code in webpack. i realize now what BOTM means, however I feel like it should be more specific so that a reader would know exactly what that stands for if they were to jump into the file with no context. Would probably be a pain to change so that can be a future fix, not one to worry about right before feature freeze.

File: webpack.config.js (hunk 1 above)
Comment (cayla-c) [reply webpack.config.js:58]: Done, thx!

File: client/dist/style.css (hunk 2 above)
Comment (cayla-c) [reply client/dist/style.css]: I think this is resolved as I am now using the style.css in the client folder.

File: client/components/homePage/BOTM.jsx (hunk 3 above)
Comment (cayla-c) [reply client/components/homePage/BOTM.jsx:5]: Agree with BOTM just didn't want to write "bookOfTheMonth" each time lol.  Have fixed webpack.  No idea what in my css could be messing with everyone else's but I will take a look now...

**Review by jfleming9357 — APPROVED** [review]

**Pull request merged**